docs/BasicCommunication/SetAppProperties/index.md

```diff
@@ -0,0 +1,105 @@
+## SetAppProperties
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Enable/disable an application and set authentication data
+
+!!! must
+
+1. Send `SetAppProperties`(enabled=true) if webEngine app is installed.
+
+!!!
+
+!!! may
+* Request `UpdateSDL` after setting app properties to enforce SDL to perform PTU.
```

Comment:
```suggestion
* Send an `[UpdateSDL](../../sd/updatesdl)` RPC after setting app properties to force SDL to perform a PTU.
```
not sure if the link works

Reply:
please, see updates in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/4ebba070bd78b95c3a00721cc2bcb639a6dfb422